cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -106,6 +107,17 @@ func clientForUserAgentOrDie(config client.Config, userAgent string) *client.Cli
 	return kubeClient
 }
 
+// clientsetForUserAgentOrDie is going to replace clientForUserAgentOrDie
+func clientsetForUserAgentOrDie(config client.Config, userAgent string) release_1_1.Interface {
```

Comment:
We'll remove `clientsetFromConfig` after the refactoring of controllers is done, which I expect we can accomplish before the 1.2 cut, and the duplication will make the cleaning easier.